src/me.rs

```diff
@@ -93,6 +94,11 @@ pub fn motion_estimation(
         steps.push(1);
       }
 
+      let x_min = 8 * (-(bo.x as isize) * MI_SIZE as isize - border_w);
```

Comment:
Would renaming these four variables to specifying the ME search range in four directions as: left_offset, right_offset, up_offset and down_offset ever confuse you? &_&